### PR TITLE
FSMA-8FSMA-88_fix_product_creation_issue

### PR DESCRIPTION
**Pull request by Deepika1095**
https://fiterio.atlassian.net/browse/FSMA-88

## Description
Describe the changes made and why they were made instead of how they were made.

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [ ] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

File: app/scripts/controllers/product/CreateLoanProductController.js [hunk 1]
```diff
@@ -314,6 +315,7 @@
 	    };
 
             scope.submit = function () {
+            console.log(this.formData);
```
Comment (Napho) [inline app/scripts/controllers/product/CreateLoanProductController.js]: I think you forgot to remove the logs

File: app/scripts/controllers/product/CreateLoanProductController.js [hunk 2]
```diff
@@ -461,6 +463,7 @@
                          this.formData.charts.push(newChart);
                       }
                 }
+                console.log(this.formData, "inend");
```
Comment (Napho) [inline app/scripts/controllers/product/CreateLoanProductController.js]: and this too

**Review by Napho — CHANGES_REQUESTED** [review]

File: app/scripts/controllers/product/CreateLoanProductController.js (hunk 1 above)
Comment (Deepika1095) [reply app/scripts/controllers/product/CreateLoanProductController.js]: ahh sorry I will do that.

**Review by oluexpert99 — DISMISSED** [review]
LGTM

**Review by Napho — APPROVED** [review]

**Review by hero78 — APPROVED** [review]

**Pull request merged**